src/systemTest/java/com/flightstats/hub/model/Channel.java

```diff
@@ -0,0 +1,26 @@
+package com.flightstats.hub.model;
+
+import lombok.Builder;
+import org.joda.time.DateTime;
+
+import java.util.Date;
+import java.util.Set;
+
+@Builder
```

Comment:
Maybe I'm missing something, but there's no ```@Data``` or ```@Value```, in which case it's all pure private values, so how does anyone get to the instance variables?

Reply:
For now these objects are not being used for reading. Only for creating for POST request body. This will evolve over time :)

Reply:
Oh! It's purely for serialization? That makes sense.

Reply:
I am not sure how we will be using this model. whether we really need it or not. It depends on the testcases. I would follow pirnciple of YAGNI here :)